recipes/nlp_primitives/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set version = "1.0.0" %}
+
+package:
+  name: nlp_primitives
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/n/nlp-primitives/nlp_primitives-{{ version }}.tar.gz
+  sha256: 5f961272e851b22268c5412333b21cc9418e3239f9fb79fd2e8e76ad1fd0436e
+
+build:
+  number: 0
+
+outputs:
+  - name: nlp-primitives
+    build:
+      noarch: python
+    script: build_script.sh
+    requirements:
+      host:
+        - pip
+        - python >=3.6
+      run:
+        - python >=3.6
+        - numpy >=1.13.3
+        - pandas >=0.23.0
+        - featuretools >=0.7.0
+        - nltk >=3.4.4
+        - scikit-learn >=0.20.0
+    test:
+      commands:
+        - pytest --pyargs nlp_primitives
+      imports:
+        - nlp_primitives
+      requires:
+        - pytest >=4.4.1
+        - tensorflow >=1.14.0
+        - tensorflow-hub >=0.4.0
+      source_files:
+        - nlp_primitives/*
+  
+  - name: nlp-primitives-complete
+    build:
+      noarch: python
+    requirements:
+      run:
+        - {{ pin_subpackage('nlp-primitives', max_pin="x.x.x.x") }}
+        - tensorflow >=1.14.0
+        - tensorflow-hub >=0.4.0
+    test:
+      commands:
+        - pytest --pyargs nlp_primitives
+      imports:
+        - nlp_primitives
+      requires:
+        - pytest >=4.4.1
+      source_files:
+        - nlp_primitives/*
+
+about:
+  doc_url: https://github.com/FeatureLabs/nlp_primitives
+  dev_url: https://github.com/FeatureLabs/nlp_primitives
+  home: https://github.com/FeatureLabs/nlp_primitives
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: An open source python library with natural language processing primitives, intended for use with Featuretools.
+
+extra:
+  recipe-maintainers:
+    - thehomebrewnerd
+    - rwedge
```

Comment:
CC @rwedge @thehomebrewnerd - are you willing to be maintainers?

Reply:
Yes!

Reply:
Yes!